Automattic/wp-calypso
# Rollback design of bundle cards on pricing page

## Proposed Changes

* Add hero images with vertically aligned product icons for desktop
* Rollback CSS changes made when adding Jetpack Starter. Ref: https://github.com/Automattic/wp-calypso/pull/76611
* Re-add CSS and HTML to handle multiple hero images for scaling purposes

Figma: 0MMYGZ7bRaHuwMF3fFS2VC-fi-3585%3A24919

## Testing Instructions

1. Click on the Calypso Green live link and go to `/pricing`
2. Make sure the product bundle cards match the design in Figma in the "Before" section of the designs
3. Check all view widths to make sure it is always readable and makes sense
![image](https://github.com/Automattic/wp-calypso/assets/65001528/7c07c078-c657-4e83-9e88-cda366d8a2eb)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/2277fd51-f671-4392-8a23-448b822b0fcd)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/a5b03310-2e41-4a8f-97f5-515f2fe70c80)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/e406fb57-cd9f-4673-88ae-2f6abe02412f)

## Pre-merge Checklist
